website/content/docs/commands/node/drain.mdx

```diff
@@ -70,9 +70,8 @@ capability.
   without being force stopped after a certain deadline.
 
 - `-ignore-system`: Ignore system allows the drain to complete without
-  stopping system job allocations. By default system jobs are stopped
-  last. You should always use this flag when draining a node running
-  [CSI node plugins][internals-csi].
+  stopping system job allocations. By default system jobs (and CSI
+  plugins) are stopped last, after the `deadline` time has expired.
```

Comment:
Note to reviewers: this "after the `deadline` time has expired" bit is existing behavior that didn't seem to be explicitly documented.